templates/utils.js

```diff
@@ -1,10 +1,20 @@
+// https://github.com/GoogleChromeLabs/jsbi/issues/30#issuecomment-1006086291
+(BigInt.prototype.toJSON = function () {
+  return this.toString();
+});
```

Comment:
Even for me whenever I write `JSON.stringify` while working on any project, My brain visually remind of this code snippet(along with that issue no.) and warns me to add it 😂

Reply:
Same here, didn't even meet BigInts while working on this )